app/helpers/locations_helper.rb

```diff
@@ -78,6 +78,12 @@ def toggle_location_link(location)
     return govuk_button_link_to "Reactivate this location", scheme_location_new_reactivation_path(location.scheme, location) if location.deactivated?
   end
 
+  def location_creation_success_notice(location)
+    if location.confirmed
+      "#{location.postcode} #{location.startdate.blank? || location.startdate < Time.zone.now ? 'has been' : 'will be'} added to this scheme"
+    end
```

Comment:
I believe the condition here can be slightly more concise `location.startdate&.before? Time.zone.now`

Reply:
how comesies not equivalent?
location.startdate type is DateTime, so surely it's either a date or nil?

Reply:
yep but we want nil to pass the ternary condition

Reply:
pleasure doing business sir

Reply:
🤝